notification-service/src/main/resources/application.yml

```
@@ -20,4 +20,9 @@ spring:
 server:
   port: 8081
 
-
+eureka:
+  client:
+    service-url:
+      defaultZone: http://localhost:19090/eureka/
+    register-with-eureka: true
+    fetch-registry: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)